src/modules/wallet/api/state.ts

```diff
@@ -30,3 +31,5 @@ export const walletDisplayedAddress = atom((get) => {
 
   return ensName || (account ? shortenAddress(account) : '')
 })
+
+export const hwAccountIndexAtom = atomWithStorage<number>('hw-account-index:v1', 0)
```

Comment:
Does this also apply for Ledger?

Reply:
Not for now, I've left TODO for `HARDWARE_WALLETS` const